.github/workflows/sphinx-build.yml

```diff
@@ -0,0 +1,59 @@
+name: Sphinx
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  documentation:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python_version: 3.8
+
+    - name: Install Dependencies
+      run: |
+        python -m pip install -U pip
+        pip install --progress-bar off -U .[document]
+
+    - name: Build Document
+      run: |
+        cd docs
+        make html -j
```

Comment:
Why did you add `-j` option?
It was ignored in the `Makefile` when I dumped the following command:

https://github.com/optuna/optuna/blob/dc032afdda0e0248bb70831ee47ca10aeca3bd28/docs/Makefile#L20

The command dump:

```console
$ make html -j
echo @sphinx-build -M html "source" "build" -W --keep-going 
@sphinx-build -M html source build -W --keep-going
```

Reply:
It's just my habit. Removed.